stubs/psycopg2/psycopg2/sql.pyi

```diff
@@ -1,48 +1,49 @@
+from collections.abc import Iterator
 from typing import Any
 
 class Composable:
     def __init__(self, wrapped) -> None: ...
     def as_string(self, context) -> str: ...
-    def __add__(self, other): ...
-    def __mul__(self, n): ...
-    def __eq__(self, other): ...
-    def __ne__(self, other): ...
+    def __add__(self, other) -> Composed: ...
+    def __mul__(self, n) -> Composed: ...
+    def __eq__(self, other) -> bool: ...
+    def __ne__(self, other) -> bool: ...
 
 class Composed(Composable):
     def __init__(self, seq) -> None: ...
     @property
-    def seq(self): ...
-    def as_string(self, context): ...
-    def __iter__(self): ...
-    def __add__(self, other): ...
-    def join(self, joiner): ...
+    def seq(self) -> list[Composable]: ...
+    def as_string(self, context) -> str: ...
+    def __iter__(self) -> Iterator[Composable]: ...
+    def __add__(self, other) -> Composed: ...
+    def join(self, joiner) -> Composed: ...
 
 class SQL(Composable):
     def __init__(self, string) -> None: ...
     @property
-    def string(self): ...
-    def as_string(self, context): ...
-    def format(self, *args, **kwargs): ...
-    def join(self, seq): ...
+    def string(self) -> str: ...
+    def as_string(self, context) -> str: ...
+    def format(self, *args, **kwargs) -> Composed: ...
+    def join(self, seq) -> Composed: ...
 
 class Identifier(Composable):
     def __init__(self, *strings) -> None: ...
     @property
-    def strings(self): ...
+    def strings(self) -> tuple[str]: ...
```

Comment:
`tuple[str]` is a tuple of length one, therefore:

```suggestion
    def strings(self) -> tuple[str, ...]: ...
```

Reply:
Fixed.